bash_scripts/generate_html_index_file.bash

```diff
@@ -83,6 +93,17 @@ ${casename}<br>(Climatological years: $begin_yr_climo-$end_yr_climo) vs. $ref_ca
 </div>
 EOF
 
+if [ $generate_ocnice_diags -eq 1 ]; then
+  cat >> index.html << EOF
+  <br>
+  <hr noshade size=2 size="100%">
+  <br>
+  <br>
+  <font color=red size=+1.5><b>Click <a href=$mpas_www_link>here</a> to view full MPAS-Analysis results (only ocn/sea-ice trends are still shown below)</b></font><br>
```

Comment:
Note that I changed this link from absolute to relative (`mpas_www_link` is simply `./mpas_analysis` at this point), so that, if we move the whole html directory somewhere else, such as at NERSC, for sharing, we won't have a broken link.